### ISSUE TITLE
deno fmt removed footnotes indent

### ISSUE THREAD

**Issue by oscarotero** [denoland/deno#25224]
In Deno 1.45 and lower it was possible to create footnotes with several paragraphs:

```md
Here is a footnote reference,[^1] and another.[^longnote]

[^1]: Here is the footnote.

[^longnote]: Here's one with multiple blocks.

    Subsequent paragraphs are indented
```

In Deno 1.46.0, the indentation is removed:

```md
Here is a footnote reference,[^1] and another.[^longnote]

[^1]: Here is the footnote.

[^longnote]: Here's one with multiple blocks.

Subsequent paragraphs are indented
```

**Comment by nathanwhit** [comment]
I think this is fixed by #25222, is that right @dsherret?

**Comment by dsherret** [comment]
Yes, fixed by https://github.com/denoland/deno/pull/25222

https://dprint.dev/playground/#code/BIUwTiAECWDOkENIDMD2qAuA7TULPBCwGMQAaAbQD0BGAXUSwBNHMALcAOmoBtUsA5jgwg6AWABQk6vQBckUBBjwMHFOmy5Ok6VT6Dho+YpAByePygB3aKsgBbAK48M0AA48oAIz7EA1rDaUhKQoZAAyo5esCAAjo5EGJBuCGAIAmlubPCpUNDMiSBMQA/plugin/markdown
